packages/python-packages/apiview-copilot/evals/tests/mention_action/update_kb.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
The main issue. Having an expanded JSON file per test case is easier to maintain and we can "zip" these up into a JSONL file for execution.